livekit_internal.proto

```diff
@@ -92,8 +92,10 @@ message NodeStats {
   float participant_join_per_sec = 27;
   uint64 participant_rtc = 44;
```

Comment:
Rename these to `participant_rtc_connected_*` to make the difference between `init` and `connected` clear please?